lib/tests/streamlit/cli_test.py

```diff
@@ -63,10 +63,19 @@ def test_run_non_existing_file_argument(self):
             "streamlit.cli._main_run"
         ), patch("os.path.exists", return_value=False):
 
-            result = self.runner.invoke(cli, ["run", "file_name"])
+            result = self.runner.invoke(cli, ["run", "file_name.py"])
         self.assertNotEqual(0, result.exit_code)
         self.assertTrue("File does not exist" in result.output)
 
+    def test_run_not_allowed_file_extension(self):
+        """streamlit run should fail if a not allowed file extension is passed"""
```

Comment:
This should end with a period.

Reply:
the majority of comments has no period, should i add it for those?